### PR TITLE
Fixing documentation for LAD Core

### PR DESCRIPTION
**Pull request by zmhassan**
# Description
Improvements to documentation

### CODE REVIEW

**Comment by zmhassan** [comment]
@4n4nd @HumairAK Please review and merge this when you can.

File: openshift/README.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# OpenShift Templates
```
Comment (HumairAK) [inline openshift/README.rst]: For .rst files you need to add a heading like: 
```
Openshift Templates
-------------
```

File: openshift/README.rst [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+The following files can be used to deploy different components of log-anomaly-detector. Below I describe what each template is used for:
+
+* aiops_factstore.deployment.yaml - Human in the loop feedback system and metadata store for LAD ML Service.
+* aiops_lad_core.buildconfig.yaml - Openshift build configuration for building s2i image from git to kubernetes container image.
```
Comment (HumairAK) [inline openshift/README.rst]: Kubernetes should be with a capital 'K'

File: openshift/README.rst [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+* aiops_lad_core.elasticsearch.storage.yaml - Elasticsearch component for usage along with LAD to process logs that come in.
+* aiops_lad_core.ml.job.yaml	 - Kubernetes job template to run LAD CORE machine learning service.
+* aiops_lad_core.k8s.service.yaml - kubernetes service to connect log anomaly detector pod to service.
```
Comment (HumairAK) [inline openshift/README.rst]: Kubernetes should be with a capital 'K'

File: openshift/README.rst [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+aiops_lad_core.minimal.deployment.yaml
+
+
+log-anomaly-detector-deployment.app.yaml
+
+aiops_lad_core.env_var_config.deployment.yaml
+
+aiops_lad_core.configmap.deployment.yaml
+aiops_lad_core.example.configuration.yaml
```
Comment (HumairAK) [inline openshift/README.rst]: These seem randomly placed here, maybe you forgot to remove them?

File: openshift/README.rst (hunk 1 above)
Comment (zmhassan) [reply openshift/README.rst]: Fixed

File: openshift/README.rst (hunk 2 above)
Comment (zmhassan) [reply openshift/README.rst]: Fixed

File: openshift/README.rst (hunk 3 above)
Comment (zmhassan) [reply openshift/README.rst]: Fixed

File: openshift/README.rst (hunk 4 above)
Comment (zmhassan) [reply openshift/README.rst]: Fixed

**Pull request merged**